samples/crawler/build.gradle.kts

```diff
@@ -1,4 +1,5 @@
 plugins {
+  kotlin("jvm")
```

Comment:
The root project was applying it to all subprojects, but this was the only one that didn‘t have it locally also.